tests/foreman/api/test_templatesync.py

```diff
@@ -955,7 +957,12 @@ def test_positive_export_json_output(
         imported_count = [
             template['imported'] for template in imported_templates['message']['templates']
         ].count(True)
-        assert imported_count == 17  # Total Count
+        exp_count = (
+            NOF_FOREMAN_TEMPLATES
+            if "puppet" in target_sat.get_features()
+            else NOF_FOREMAN_TEMPLATES - 1
```

Comment:
this feels hardcody. maybe define the constant as a dict to hold both vals for each scenario.
```
FOREMAN_TEMPLATES_IMPORT_COUNT = {
    'PUPPET_ENABLED': 18,
    'PUPPET_DISABLED': 17,
}
```
or possibly even expand it to contain the "not_imported" or "skipped" counts.